src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.Person;
+
+import java.util.List;
+
+public class SortCommand extends Command {
```

Comment:
No header comment.

Reply:
OK I will take note in feature and remember that every non-trivial class and non-trivial method inside a class requires a header comment whenever possible :)